views/thingpedia_dev_sidebar.pug

```diff
@@ -40,11 +40,14 @@ nav#doc-sidebar
             li(class=(currentPage === 'thingpedia-tutorial-cat' ? 'current' : ''))
               a(href='/thingpedia/developers/thingpedia-tutorial-cat.md') Tutorial 2: The Cat API
 
+            li(class=(currentPage === 'thingpedia-tutorial-website-monitor' ? 'current' : ''))
+              a(href='/thingpedia/developers/thingpedia-tutorial-website-monitor.md') Tutorial 3: Class Website Monitor
+
             li(class=(currentPage === 'thingpedia-tutorial-linkedin' ? 'current' : ''))
-              a(href='/thingpedia/developers/thingpedia-tutorial-linkedin.md') Tutorial 3: LinkedIn
+              a(href='/thingpedia/developers/thingpedia-tutorial-linkedin.md') Tutorial 4: LinkedIn
```

Comment:
I would put LinkedIn before this one, because this one introduces zip files, dependencies, and so on.